test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -4303,11 +4305,41 @@ def _load_configuration(self, infile, objtype=None):
                             self.logger.error("Failed to restore "
                                               + "configuration: %s" % k)
                             return False
+                        rv = self.du.chown(path=k, runas=ROOT_USER,
+                                           uid=0, gid=0, sudo=True)
+                        if not rv:
+                            self.logger.error("Failed to restore "
+                                              + "configuration: %s" % k)
+                            return False
                     finally:
                         if os.path.isfile(fn):
-                            self.du.rm(path=fn)
+                            self.du.rm(path=fn, force=True, sudo=True)
+                return True
+            elif objtype == MGR_OBJ_NODE:
+                conf = sconf[str(objtype)]
+                nconf = {}
```

Comment:
I think the above initialization is not needed since you are assigning the value in the next line.